File: 3285.find-indices-of-stable-mountains.py
#
# @lc app=leetcode id=3285 lang=python
#
# [3285] Find Indices of Stable Mountains
#

# @lc code=start
class Solution(object):
    def stableMountains(self, height, threshold):
        """
        :type height: List[int]
        :type threshold: int
        :rtype: List[int]
        """
        r = []
        for i in range(1, len(height)):
            if height[i-1] > threshold:
                r.append(i)
        return r
# @lc code=end

File: 3285.find-indices-of-stable-mountains.java
/*
 * @lc app=leetcode id=3285 lang=java
 *
 * [3285] Find Indices of Stable Mountains
 */

// @lc code=start
class Solution {
    public List<Integer> stableMountains(int[] height, int threshold) {
        ArrayList<Integer> r = new ArrayList<>();
        for (int i = 1; i < height.length; i++) {
            if (height[i-1] > threshold) {
                r.add(i);
            }
        }
        return r;
    }
}
// @lc code=end

